website/static/js/registrationUtils.js

```diff
@@ -263,7 +263,8 @@ var MetaSchema = function(params) {
     $.each(self.schema.pages, function(i, page) {
         var mapped = {};
         $.each(page.questions, function(qid, question) {
-            mapped[qid] = new Question(question, qid);
+            var questionId = question.qid;
```

Comment:
Wait, questions is now a list-- so we need to:
`$.each(page.questions, function(qid, question) {`
to
`$.each(page.questions, function(index, question) {`
and leave L266-7 as is